src/server/metadata/cmds/cmds_test.go

```diff
@@ -202,14 +202,87 @@ func TestParseEditMetadataCmdline(t *testing.T) {
 			args:    []string{"project", "default", "set", "key=value"},
 			wantErr: true,
 		},
+		{
+			name: "edit a commit",
+			args: []string{"commit", "images@master", "edit", "key=value",
+				"commit", "images@44444444444444444444444444444444", "edit", "key=value"},
+			want: &metadata.EditMetadataRequest{
+				Edits: []*metadata.Edit{
+					{
+						Target: &metadata.Edit_Commit{
```

Comment:
the nesting ... 🤯

Reply:
It's not so bad in JSON: 
```json
{
  "branch_head": {
    "name": {
      "repo": {
        "name": {
          "project": {
            "name": "default"
          },
          "name": "images",
          "type": "user"
        }
      },
      "name": "master"
    }
  },
}
```
Only suggestion is that maybe we want to call the fields that use name pickers "by_name" instead?